src/solver.h

```diff
@@ -208,6 +208,8 @@ struct s_Solver {
   Map allowuninstallmap;		/* ok to uninstall those */
   int allowuninstall_all;
 
+  Map weakexcludemap;		/* Do not use them for supplements and recommends */
```

Comment:
The comment doesn't make sense to me.
Don't we want to actually use weak excludes on Supplements and Recommends?

Reply:
I would expect that we'd want to, yes...